Library/Homebrew/lock_file.rb

```diff
@@ -51,14 +53,24 @@ def create_lockfile
 
 # A lock file for a formula.
 class FormulaLock < LockFile
-  def initialize(name)
-    super("#{name}.formula")
+  sig { params(rack_name: String).void }
+  def initialize(rack_name)
+    super(:formula, HOMEBREW_CELLAR/rack_name)
   end
 end
 
 # A lock file for a cask.
 class CaskLock < LockFile
-  def initialize(name)
-    super("#{name}.cask")
+  sig { params(cask_token: String).void }
+  def initialize(cask_token)
+    super(:cask, HOMEBREW_PREFIX/"Caskroom/#{cask_token}")
```

Comment:
```suggestion
    super(:cask, HOMEBREW_CASKROOM/cask_token)
```

Reply:
That constant isn't actually defined globally 😭